plugins/package-managers/bazel/src/main/kotlin/Bazel.kt

```diff
@@ -82,9 +82,7 @@ class Bazel(
 
     override fun command(workingDir: File?) = "bazel"
 
-    override fun getVersionArguments() = "version"
-
-    override fun transformVersion(output: String) = transformBazelVersion(output)
+    override fun transformVersion(output: String) = output.removePrefix("bazel ")
```

Comment:
Nit: Maybe say "Using the *default* `bazel --version`..." to better explain why this work by basically just removing code.